README.md

```diff
@@ -56,6 +56,68 @@ jobs:
       - run: java -cp java HelloWorldApp
 ```
 
+Publishing using Apache Maven:
+```yaml
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+        server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
+        username: ${{ github.actor }} # username for server authentication
+        password: ${{ github.token }} # password or token for authentication
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
+    - name: Publish to GitHub Packages Apache Maven
+      run: mvn deploy
+    - name: Set up Apache Maven Central
+      uses: actions/setup-java@v1
+      with: # running setup-java again overwrites the settings.xml
+        java-version: 1.8
+        server-id: maven
+        username: maven_username
+        password: ${{ secrets.MAVEN_CENTRAL_TOKEN }} # password from secrets store
+    - name: Publish to Apache Maven Central
+      run: mvn deploy 
+```
+
+***NOTE: The `settings.xml` is created in the Actons $HOME directory by default and will overwrite existing files.***	
```

Comment:
```suggestion
***NOTE: The `settings.xml` file is created in the Actions $HOME directory. If you have an existing `settings.xml` file at that location, it will be overwritten***	
```
Small spelling fix. Also some rewording. Sounds a little scary right now as it seems like all existing files will be overwritten.